.devcontainer/Dockerfile

```
@@ -20,6 +31,11 @@ RUN if [ -z "${USERNAME}" ] ; then exit 1 ; fi \
 
 RUN install -d -m 0755 -o ${USERNAME} -g ${USERNAME} /workspace
 
+COPY --from=latest-go-fetcher /tmp/go /tmp/go
+RUN GOPATH=$(dirname $(dirname $(which go))) \
```

Hadolint (2.12.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)